app/services/seed/tpi_data.rb

```diff
@@ -0,0 +1,48 @@
+require "#{Rails.root}/lib/timed_logger"
+
+module Seed
+  class TPIData
+    include Singleton
+
+    class << self
+      delegate :call, to: :instance
+    end
+
+    # rubocop:disable Metrics/AbcSize
+    def call
+      # import NewsArticles
+      TimedLogger.log('Import news articles') do
```

Comment:
I'm going to move the NewsArticles out of this main task. As they already started looking at them in prod, so wouldn't want to risk removing it.

Reply:
Actually I'll just comment it for now. won't need to run them in prod.

Reply:
Should be actually moved out of this tpi_data class back to seeds.rb

Reply:
yeah, we can probably get rid of it completely soon. Or maybe once we "lock up" the seeds to not run in prod. As we won't need to run this there, we can keep it for the development env later. I'll move it.